middlewares/error-handler.js

```diff
@@ -11,6 +11,5 @@ module.exports = (err, req, res, next) => {
     res.status(err.statusCode).json({ status: 'error', message: err.message });
     return;
   }
-  console.error(err);
```

Comment:
why delete the line? when server have some error that does't know. server should send to log.

Reply:
Sorry, I thought it was me who added this line, because I added a lot of console.logs to understand the problem. I will add it again